src/scss/shared/_control-inputs.scss

```diff
@@ -42,7 +42,7 @@
 	float: left;
 	margin: 0;
 	min-height: $_o-forms-spacing-seven;
-	min-width: oTypographySpacingSize(15);
+	min-width: 8ch;
```

Comment:
is this width not to be based on what the spacing widths are based on?

Reply:
It's less typical to use them for widths, usually that's left to the content length or a containing element. In this case ([box style radio inputs](https://registry.origami.ft.com/components/o-forms@7.0.9#demo-box-styled-radio-inputs))  most of the time the length of the content in the box will set the width, so I switched the min-width out to also be derived from the width of the content.